src/components/Homepage/Homepage.css

```
@@ -3,12 +3,11 @@
   --btn-color: #da1b24;
 }
 .Homepage_heading_Container {
```

`
```suggestion
.Homepage_HeadContainer {
```

Adding more than one `_` would make our classes very long. Only exception is when you are creating a variant.